deployments/kubernetes/install/scripts/install-cni.sh

```diff
@@ -78,6 +78,8 @@ ${CNI_NETWORK_CONFIG}
 EOF
 fi
 
+KUBECFG_FILE_NM=${KUBECFG_FILE_NM:-ZZZ-istio-cni-kubeconfig}
```

Comment:
nit: I'd prefer `KUBECFG_FILE_NAME` over `KUBECFG_FILE_NM`, just to keep it consistent with the rest of the variable names.

Reply:
will do